build/multilocale.js

```diff
@@ -260,6 +261,10 @@ function L10nManager(gaiaDir,
       metas.defaultLanguage = createMeta(doc, 'defaultLanguage');
     }
 
+    if (!metas.gaiaVersion) {
+      metas.gaiaVersion = createMeta(doc, 'gaiaVersion');
```

Comment:
Since we're getting rid of the Gecko2Gaia mapping and we're becoming Gecko-agnostic, could we call this `appVersion` and become Gaia-agnostic as well?